### PR TITLE
fix(import): RHICOMPL-3331 import selected groups correctly

### PR DESCRIPTION
**Pull request by marleystipich2**
This PR relies on this [openscap_parser PR](https://github.com/OpenSCAP/openscap_parser/pull/39) being merged and released but I tested locally with the gem updates and the import worked correctly and the tests passed. 

To get the tests to pass locally: 
Checkout this branch locally and then update the openscap_parser line in the Gemfile to:
`gem 'openscap_parser', git: 'https://github.com/OpenSCAP/openscap_parser.git', branch: 'master'`

(Tests will continue to fail with `NoMethodError: undefined method `unselected_group_ids' `until openscap_parser changes are released)

## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [x] General Coding Practices

### CODE REVIEW

**Review by romanblanco — APPROVED** [review]
LGTM :+1:

**Comment by skateman** [comment]
How does it look like time-wise in ephemeral? Is this slower/faster?

**Comment by romanblanco** [comment]
Should be tested before merging. The code looks good.

**Comment by marleystipich2** [comment]
@skateman @romanblanco I deployed the master branch to ephemeral and the total deployment time was 9 minutes 36 seconds and then I deployed this branch to ephemeral and the total deployment time was 9 minutes. The `compliance-import-ssg` job took 6 minutes for the master branch and 4 minutes for this branch. 
I thought that these changes would make the import take a little longer though because we're importing more data 🤔 

If this sounds good to you guys though, we can go ahead and merge :)

**Comment by skateman** [comment]
You forgot the migration to reset revisions in order to force the SSG reimport without setting the environment variable.

**Comment by marleystipich2** [comment]
@skateman Sorry, I forgot about that! Just added it. I'll merge if it looks good to you guys now.

**Review by skateman — APPROVED** [review]

**Pull request merged**
